src/map_object_move.c

```diff
@@ -813,21 +809,21 @@ static const int DATA_GPosY_Dir4AddTbl[] = {
     0
 };
 
-static const int sMapObjectDyDir[] = {
+static const int sMapObjectDzDir[] = {
     -1,
     0x1,
     0x0,
     0x0
 };
```

Comment:
Similar suggestion applies to `sMapObjectDxDir` above.

```suggestion
static const int sMapObjectDzDir[] = {
    [DIR_NORTH] = -1,
    [DIR_SOUTH] = 1,
    [DIR_WEST] = 0,
    [DIR_EAST] = 0,
};
```

Reply:
```static const int DATA_GPosY_Dir4AddTbl[] = {
    0,
    0,
    0,
    0
};```
This is in the middle of those two, and it is unused. What's the best way to go about that? Remove it, rename it to something like `UNUSED_GPosY_Dir4AddTbl`, or add a comment like `// unused`?

Reply:
Something like that would be fine, yeah!

Reply:
Done!